src/jit/compiler.h

```diff
@@ -5873,6 +5873,13 @@ class Compiler
 
     typedef unsigned short AssertionIndex;
 
+    enum optAssertionEdge : AssertionIndex
```

Comment:
Would be nice to have a comment here indicating that the assertion index on a JTRUE is the bitwise-or of the appropriate optAssertionEdge and its normal index.  Nice to have a comment about that over on the definition of the assertion field/accessor, too.